deployment/chart/values.yaml

```diff
@@ -71,25 +71,43 @@ tolerations: []
 
 affinity: {}
 
-ticketsEnvironment: "production"
-ticketsSecretToken: "this-is-a-very-bogus-key"
-ticketsServeStaticFiles: "true"
-ticketsSMTPHost: ""
-ticketsSMTPPort: "" 
-ticketsSMTPDomain: ""
-ticketsSMTPUser: ""
-ticketsSMTPPassword: ""
-ticketsDBSeed: ""
-ticketsHost: ""
-ticketsEmailAddress: ""
-stripeSecretKey: ""
-stripePublicKey: ""
-pumaConcurrency: 2
-pumaThreads: 2
+config:
+  ticketsEnvironment: "production"
+  ticketsSMTPHost: ""
+  ticketsSMTPPort: "" 
+  ticketsSMTPDomain: ""
+  ticketsDBSeed: ""
+  ticketsHost: ""
+  ticketsEmailAddress: ""
+  stripePublicKey: ""
```

Comment:
We may not need to keep any of these in the helm chart except Rails Master key.

This is because we can now encrypt the credentials for all environments and it can be pushed to GitHub. 

So stripe key, smtp credentials are all defined in config/credentials.yml.enc

As long as we place the encryption master key into `config/master.key` we will automatically decrypt all the sensitive values

Reply:
I believe we have decided to put all secrets in GCP SecretManager which is better all around. Just making sure we are still on the same page regarding this

Reply:
What i am sating is the only things we are going to put into that secret manager will be:

- rails secret called master.key. It should set environment `RAILS_MASTER_KEY` before invoking the app container
- Database credentials and/or URL
- anything else needed not for the application but for the rest of the pod